consumerBuilder.go

```diff
@@ -258,6 +259,19 @@ func (c *consumerBuilder) getOrAddSaramaConsumer(cluster string, topicList []con
 	return saramaConsumer, nil
 }
 
+func (c *consumerBuilder) uniqueTopics(topics []consumer.Topic) []consumer.Topic {
+	topicSet := make(map[string]consumer.Topic)
```

Comment:
nit: do we need to copy consumer.Topic since we only use it as a set with topic name as key anyways

Reply:
Good point. Fixed.